src/Tizen.NUI/src/public/BaseComponents/CameraView.cs

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using Tizen.NUI.Binding;
+
+namespace Tizen.NUI.BaseComponents
+{
+    /// <summary>
+    /// CameraView is a view for camera preview.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class CameraView : View
+    {
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public enum DisplayType
+        {
+            Window = 0,   //  HW overlay
+            Image     //  texture stream
+        };
+
+        /// <summary>
+        /// Creates an initialized CameraView.
+        /// </summary>
+        /// <param name="handle">Multimedia Camera handle</param>
+        /// <param name="type">DisplayType</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public CameraView(global::System.IntPtr handle, DisplayType type = DisplayType.Window) : this(Interop.CameraView.New(), true)
```

Comment:
as you know, raw pointer (IntPtr) is not supposed to be exposed in public API. 
this needs to be changed as using `SafeHandle`. (https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/BaseComponents/VideoView.cs#L502)
currently, this is hidden-API so it would be OK. but please fix when this is published by ACR.
thank you.